pond.go

```diff
@@ -318,8 +318,13 @@ func (p *WorkerPool) SubmitBefore(task func(), deadline time.Duration) {
 // Stop causes this pool to stop accepting new tasks and signals all workers to exit.
 // Tasks being executed by workers will continue until completion (unless the process is terminated).
 // Tasks in the queue will not be executed.
```

Comment:
Shall we add a small comment to explain the returned `context.Context`? Feel free to adjust it if it's not clear enough.
```suggestion
// Stop causes this pool to stop accepting new tasks and signals all workers to exit.
// Tasks being executed by workers will continue until completion (unless the process is terminated).
// Tasks in the queue will not be executed.
// This method returns a context object that is cancelled when the pool has stopped completely.
```

Reply:
You are right, that looks good